product_docs/docs/edbcloud/beta/reference/index.mdx

```diff
@@ -205,10 +210,63 @@ The response of this API call includes the `access_token`. For example:
 
 
 
+## Using the get-token Script
```

Comment:
Should we put "get-token" in `monospace` everywhere in this document? It reads a bit strangely to me as plain text, but I don't know the prevailing style well.